app/bundles/LeadBundle/Controller/LeadController.php

```diff
@@ -1581,6 +1580,94 @@ private function resetImport($filepath)
         unlink($filepath);
     }
 
+    public function exportCurrentListAction(){
```

Comment:
Does it make more sense to integrate with existing export functionality than to write an entirely new CSV export function?